hash/pc98_flop_misc.xml

```diff
@@ -3,7 +3,7 @@
 <!--
 license:CC0-1.0
 -->
-<softwarelist name="pc98" description="NEC PC-9801 disk images">
+<softwarelist name="pc98_flop_misc" description="NEC PC-9801 miscellaneous disk images">
```

Comment:
Not a fan of using `misc`, in data context it has absolutely no value. `Unverified` / `Legacy` or even just `Cracked` would be better options, while filename can just be `pc98_flop.xml` (given that we won't get rid of this soon anyway).

Reply:
Done. I went with "unverified" since that's basically what it contains, images that haven't been compared with original disks, and not all of them are necessarily cracked.